### ISSUE TITLE
Engine panics when schema.prisma is empty, we could make it an error.

### ISSUE THREAD

**Issue by Jolg42** [prisma/prisma#3376]
How to reproduce?

- in latest dev version
- create an empty `prisma/schema.prisma` file 
- use `prisma generate`

<img width="790" alt="Screen Shot 2020-08-21 at 15 46 06" src="https://user-images.githubusercontent.com/1328733/90897830-011e1580-e3c6-11ea-8fe7-26ff12fa93f2.png">

**Comment by mavilein** [comment]
The culprit is very likely this line: https://github.com/prisma/prisma-engines/blob/master/libs/datamodel/core/src/configuration/configuration.rs#L14

We recently added validation that one datasource must be present when you e.g. start the Query Engine. All our errors need a `Span`. But in this case there's no span available because the error is about an element that does not exist. So i hard coded `(0,5)` when i implemented it. I chose a length of 5 because i wanted a squiggly line to be somewhat visible. During the rendering of the error (putting the nice underlines and such) this crashes because the original input is less long in this case.
Maybe we should try this with `(0,0)` instead and see how the error looks like then. Btw the error report of @Jolg42 indicates that this happens during `prisma generate`. But we don't perform the validation anymore in this case. The only way to test this right now is by starting a QE / instantiating a Prisma Client.
